doc/src/arch/reference.rst

```diff
@@ -1897,6 +1902,98 @@ The clocking configuration is specified with ``<clock>`` tags within the ``<cloc
     :opt_param C_wire_per_m: The wire capacitance, in Farads per Meter.
     :opt_param buffer_size: The size of each clock buffer.
 
+**Specifing a Clock Architecture**
+
+The element ``<clocknetworks>`` contains three sub-elements that collectively describe the clock architecture: the wiring parameters ``<metal_layers>``, the clock distribution ``<clock_network>``, and the clock connectivity ``<clock_routing>``. An example is shown below:
+
+    .. code-block:: xml
+    <clocknetworks>
+        <metal_layers >
+            <metal_layer name="global_spine" Rmetal="50.42" Cmetal="20.7e-15"/>
+            <metal_layer name="global_rib" Rmetal="50.42" Cmetal="20.7e-15"/>
+        </metal_layers >
+
+        <!-- Full Device: Center Spine -->
+        <clock_network  name="spine1" num_inst="2">
+            <spine metal_layer="global_spine"12 x="W/2" starty="0" endy="H">
+                <switch_point type="drive" name="drive" yoffset="H/2" buffer="drive_buff"/>
+                <switch_point type="tap" name="tap" yoffset="0" yincr="1"/>
+            </spine>
+        </clock_network>
+        <!-- Full Device: Each Grid Row -->
+        <clock_network  name="rib1" num_inst="2">
+            <rib metal_layer="global_rib" y="0" startx="0" endx="W" repeatx="W" repeaty="1">
+                <switch_point type="drive" name="drive" xoffset="W/2" buffer="drive_buff"/>
+                <switch_point type="tap" name="tap" xoffset="0" xincr="1"/>
+            </rib>
+        </clock_network>
+
+        <clock_routing>
+            <!-- connections from inter-block routing to central spine -->
+            <tap from="ROUTING" to="spine1.drive" locationx="W/2" locationy="H/2" switch="general_routing_switch" fc_val="1.0"/>
+            <!-- connections from spine to rib -->
+            <tap from="spine1.tap" to="rib1.drive" switch="general_routing_switch" fc_val="0.5"/>
+            <!-- connections from rib to clock pins -->
+            <tap from="rib1.tap" to="CLOCK" switch="ipin_cblock" fc_val="1.0"/>
+        </clock_routing >
+    </clocknetworks >
+
+The ``<metal_layers>`` element describes the per unit length electrical parameters used to implement the clock distribution  wires. There can be one or more wiring implementation options (metal layer, width and spacing) that are used by the later  clock network specification and each is described in a separate ``<metal_layer>`` sub-element. The syntax of the wiring electrical information is:
```

Comment:
I'd likely add that the metal layer width and spacing are the physical implementation characteristics which result in the Rmetal/Cmetal but that we only model them as those characteristics (as currently written someone might expect width/spacing options for the metal layers).